src/test/java/uk/gov/hmcts/reform/cpo/auditlog/aop/AuditAspectTest.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.reform.cpo.auditlog.aop;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.aop.aspectj.annotation.AspectJProxyFactory;
+import org.springframework.aop.framework.AopProxy;
+import org.springframework.aop.framework.DefaultAopProxyFactory;
+import org.springframework.stereotype.Controller;
+import uk.gov.hmcts.reform.BaseTest;
+import uk.gov.hmcts.reform.cpo.auditlog.AuditOperationType;
+import uk.gov.hmcts.reform.cpo.auditlog.LogAudit;
+import uk.gov.hmcts.reform.cpo.domain.CasePaymentOrder;
+import uk.gov.hmcts.reform.cpo.payload.UpdateCasePaymentOrderRequest;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.springframework.test.util.AssertionErrors.assertEquals;
+import static org.springframework.test.util.AssertionErrors.assertNotNull;
+import static org.springframework.test.util.AssertionErrors.assertNull;
+import static org.springframework.test.util.AssertionErrors.assertTrue;
+
+class AuditAspectTest implements BaseTest {
+
+    private final AuditAspect aspect = new AuditAspect();
+    private TestController controllerProxy;
+
+    @BeforeEach
+    void setUp() {
+        AspectJProxyFactory aspectJProxyFactory = new AspectJProxyFactory(new TestController());
+        aspectJProxyFactory.addAspect(aspect);
+
+        DefaultAopProxyFactory proxyFactory = new DefaultAopProxyFactory();
+        AopProxy aopProxy = proxyFactory.createAopProxy(aspectJProxyFactory);
+
+        controllerProxy = (TestController) aopProxy.getProxy();
+    }
+
+    @Test
+    @DisplayName("Should populate audit context")
+    void shouldPopulateAuditContext() {
+
+        // WHEN
+        CasePaymentOrder result = controllerProxy.createCpo();
+        AuditContext context = AuditContextHolder.getAuditContext();
+
+        // THEN
+        assertNotNull("Context should exist", context);
+        assertEquals("OperationTyp should be populated as per LogAudit",
```

Comment:
typo `OperationType`

Reply:
Resolved in 8fcbda9faaefab2c281ae0d6bc0cee8f12cbcac2. I repeated the same typo in many places so hopefully I have updated them all. Thanks